src/Authoring/AuthoringWinUITest/AuthoringWinUITest/Directory.Build.targets

```diff
@@ -0,0 +1,15 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <PrepareForRunDependsOn>CopyTestAssets;$(PrepareForRunDependsOn)</PrepareForRunDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CopyTestAssets">
```

Comment:
comment temporary until nuget supports (and tests like this should pull in that support as if the nuget was referenced)